Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -775,6 +774,10 @@ final class FileManagerTests : XCTestCase {
         #if !os(watchOS) && !os(tvOS)
         assertSearchPaths([.trashDirectory], exists: (isMacOS && isFramework) || (!isDarwin && !isWindows))
         #endif
+
+#if !os(Windows)
+        assertSearchPaths([.picturesDirectory], exists: true)
```

Comment:
Could we add a comment here saying that it's not tested on Windows because the directory doesn't exist in CI but likely exists on desktop machines? Just to make it clearer why it's not written as `assertSearchPaths([.picturesDirectory], exists: !isWindows)`

Reply:
Ugh, I failed to save the changes for that. I had the following comment:

```
        // .picturesDirectory does not exist in CI, though it does exist in user
        // desktop scenarios.
```